examples/vanilla-ts-esm/public/mux-uploader-full.html

```diff
@@ -102,7 +102,7 @@ <h2>Enter your upload GCS url:</h2>
           <input type="text" class="direct-upload-url" placeholder="https://storage.googleapis.com/..." />
         </section>
         <div class="button-container">
-          <mux-uploader id="uploader" type="bar" status>
+          <mux-uploader id="uploader" type="bar">
```

Comment:
nitpick(blocking): `type` is no longer a functional attribute on `<mux-uploader>`. Remove (and all other cases).